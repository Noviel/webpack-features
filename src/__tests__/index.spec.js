import * as publicAPI from '../index.js';

describe('index', () => {
  it('should expose public API', () => {
    expect(publicAPI).toMatchSnapshot();
  });
});
