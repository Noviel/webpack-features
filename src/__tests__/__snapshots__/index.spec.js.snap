// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should expose public API 1`] = `
Object {
  "default": [Function],
  "pluginEmotion": [Function],
  "presetReact": [Function],
  "tsConfigFile": [Function],
}
`;
